### DISCUSSION TITLE
Readme and License

### DISCUSSION THREAD

**Discussion by d-kleine** [rasbt/MachineLearning-QandAI-book#14] [Ideas]
Hi,

With reference to the repository, can you please add
- some more information to the _README.md_ how to make your code reproducible (such as a _requirements.txt_ or _environment.yml_)
- a license

Thank you!

**Comment by rasbt** [comment]
Great suggestion @d-kleine ! I'll put that on my list and will add it in the next couple of days!

**Reply by d-kleine** [reply]
Thanks! 👍

**Reply by rasbt** [reply]
Thanks again for the suggestions @d-kleine. I added a code license (I literally called it CODE_LICENSE.txt to avoid potential confusions with book licenses as many people reached out about translations rights :P).

There is now a `requirements.txt` file. Personally, I use conda and an `environment.yml` would be the first choice, but a `requirements.txt` is more universal for those who don't use conda.

**Reply by d-kleine** [reply]
Thank you! Yeah, I agree, a `requirements.txt` file is better 👍
